plugins/rbac-backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-rbac-backend",
-  "version": "4.10.6",
+  "version": "4.12.2",
```

Comment:
From my understanding, we don't need this change. Changesets should make this change for us after we merge this pr

Reply:
Ok, I removed this change. So next version for RHDH 1.3.1 will be 4.10.7?